### PR TITLE
automatically generate issues w/ labels for a viz

### PR DESCRIPTION
**Pull request by lindsaycarr**
`initializeVizRepo(repo_name="myviz", description="This is a description of my viz")` will create a new repo  within USGS-VIZLAB with 6 new issues with corresponding labels.

Tests for this are commented out (and will remain that way) because there is interactive authentication, plus we don't want to create and delete a repository every time the tests are run. The commented out tests will remain there for us to run manually if we need to.

See #144.

### CODE REVIEW

File: R/initializeVizRepo.R [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+#' 
+#' @return URL for the new GitHub repository.
+#' @export
+initializeVizRepo <- function(repo_name, description, org="USGS-VIZLAB",
+                              issue_json=file.path("inst", "issuetemplates.json"),
+                              label_json=file.path("inst", "labeltemplates.json")){
+  
+  # log in to GitHub & scope to read/write access for repos using profile.yaml
+  # for additional scope info, see https://developer.github.com/v3/oauth/#scopes
+  github_creds <- getProfileInfo()
```
Comment (jiwalker-usgs) [inline R/initializeVizRepo.R]: may want to verify that creds are not null here for a warning with helpful hint, error from github will likely not help too much in telling users to edit their profile.yaml

File: R/initializeVizRepo.R [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+#' @param repo_name string, name for the new repository
+#' @param description string, short description of the repository
+#' @param org string, GitHub organization to create repository. Defaults to "USGS-VIZLAB"
+#' @param issue_json file path indicating the JSON file to be used to define what issues to create
+#' @param label_json file path indicating the JSON file to be used to define what labels to create
+#' 
+#' @return URL for the new GitHub repository.
+#' @export
+initializeVizRepo <- function(repo_name, description, org="USGS-VIZLAB",
+                              issue_json=file.path("inst", "issuetemplates.json"),
```
Comment (jiwalker-usgs) [inline R/initializeVizRepo.R]: I think the defaults are left from testing, wasn't this supposed to be system.file?

Comment (lindsaycarr) [inline R/initializeVizRepo.R]: whoops - correct! I updated the test I was working on but not this function.

File: R/initializeVizRepo.R (hunk 1 above)
Comment (lindsaycarr) [inline R/initializeVizRepo.R]: Good idea

**Comment by eread-usgs** [comment]
Sorry for the delayed review- the list looks good to me @lindsaycarr

**Pull request merged**